examples/scripts/ct_multi_cs_tv_admm.py

```diff
@@ -98,7 +110,18 @@
     # Run the solver.
     solver.solve()
     hist[p] = solver.itstat_object.history(transpose=True)
-    x_rec[p] = snp.clip(solver.x, 0, 1.0)
+    x_rec[p] = solver.x
+
+    if p == "scico":
+        x_rec[p] = x_rec[p] * det_spacing  # to match ASTRA's scaling
```

Comment:
Since this is only required for the `scico` projector, it would seem to violate a common convention adopted by both `astra` and `svmbir`. Would it not make sense to include the scaling in the projector?

Reply:
Let's postpone dealing with this until later.